mon-pix/tests/unit/utils/resize-image-test.js

```diff
@@ -47,32 +47,63 @@ module('Unit | Utility | Resize Image', function () {
       assert.deepEqual(imageInformation, dimensions);
     });
 
-    test('should return accurate result when MAX_HEIGHT option is provided', function (assert) {
-      // given
-      const imageInformation = { height: 50, width: 100 };
-      const options = {
-        MAX_HEIGHT: 100,
-      };
-
-      // when
-      const dimensions = resizeImage(imageInformation, options);
-
-      //then
-      assert.deepEqual(dimensions, { height: 100, width: 200 });
+    module('when provided image is raster', function () {
+      test('should return current height when MAX_HEIGHT option is provided', function (assert) {
+        // given
+        const imageInformation = { height: 50, width: 100, type: 'raster' };
+        const options = {
+          MAX_HEIGHT: 100,
+        };
+
+        // when
+        const dimensions = resizeImage(imageInformation, options);
+
+        //then
+        assert.deepEqual(dimensions, { height: 50, width: 100 });
+      });
+      test('should return current width if MAX_WIDTH option is provided and larger than width', function (assert) {
+        // given
+        const imageInformation = { height: 100, width: 50, type: 'raster' };
+        const options = {
+          MAX_WIDTH: 100,
+        };
+
+        // when
+        const dimensions = resizeImage(imageInformation, options);
+
+        //then
+        assert.deepEqual(dimensions, { height: 100, width: 50 });
+      });
     });
 
-    test('should return current width if MAX_WIDTH option is provided and larger than width', function (assert) {
-      // given
-      const imageInformation = { height: 100, width: 50 };
-      const options = {
-        MAX_WIDTH: 100,
-      };
-
-      // when
-      const dimensions = resizeImage(imageInformation, options);
-
-      //then
-      assert.deepEqual(dimensions, { height: 100, width: 50 });
+    module('when provided image is vectoriel', function () {
+      test('should return accurate result when MAX_HEIGHT option is provided', function (assert) {
+        // given
+        const imageInformation = { height: 50, width: 100, type: 'vectoriel' };
```

Comment:
```suggestion
        const imageInformation = { height: 50, width: 100, type: 'vectorial' };
```

ou `vector` apparemment en anglais

Reply:
c'est apparemment [vector](https://github.com/1024pix/pix/blob/53932ea91bd5b6e239c7dc4424e7fe822eabf9d6/api/src/shared/domain/models/PixAssetImageInfos.js#L26) 😄